app/src/main/java/moe/apex/rule34/preferences/Pref.kt

```diff
@@ -452,11 +453,9 @@ enum class ImageSource(override val description: String, val site: ImageBoard) :
     R34("Rule34", Rule34);
 
     companion object {
-        fun get(name: String): ImageSource? {
-            return ImageSource.entries.find { it.name == name }
-        }
+        fun uriToImageView(uri: Uri?): ImageView? {
```

Comment:
Move this out of `ImageSource` into `ImageView`'s companion object. Something like `ImageView#fromUri` (also I don't think we really want to allow null values to be passed in and should handle them appropriately elsewhere)